doc/general_documentation/flight_release.md

```diff
@@ -0,0 +1,89 @@
+\page release Creating a Flight Release
+
+# Creating a Flight Release
+
+This page describes the steps required to complete a flight release. It assumes
+that your environment is configured to cross-compile the fsw for the `armhf`
+architecture.
+
+## Create a Release Branch
```

Comment:
Could we use the github release feature instead or in addition to this? 
https://github.com/nasa/astrobee/releases
That becomes more accessible for outside users and all the versions become easily tractable on the releases page.

Reply:
Actually this shouldn't be here, but in the end on the "reviewers" section after everything is tested

Reply:
I like it! (and this will also create a tag)